certbot/main.py

```diff
@@ -172,7 +172,7 @@ def _handle_identical_cert_request(config, lineage):
         return "reinstall", lineage
     question = (
         "You have an existing certificate that contains exactly the same "
-        "domains you requested and isn't close to expiry."
+        "domains or certificate name you requested and isn't close to expiry."
```

Comment:
If you're feeling fancy, provide a question whose wording varies depending on whether the user actually provided domains or a certificate name (or if you agree that's a good idea but don't want to do it in this PR, open a new ticket for it).